languages/python/sqlalchemy-oso/tests/test_flask.py

```diff
@@ -56,14 +64,16 @@ def test_authorized_sqlalchemy(ctx, flask_app, oso, sqlalchemy, post_fixtures):
 
     assert authorized_session.query(Post).count() == 1
 
-    with flask_app.app_context():
-        assert sqlalchemy.session.query(Post).count() == 1
+    assert sqlalchemy.session.query(Post).count() == 1
 
 
-def test_flask_model(ctx, flask_app, oso, sqlalchemy):
+def test_flask_model(ctx, oso, sqlalchemy):
     class TestModel(sqlalchemy.Model):
         id = Column(Integer, primary_key=True)
```

Comment:
The wowhack is necessitated by the fact that we're retrieving the `AuthorizedSQLAlchemy` instance's base model (`Model` field), which is only available _after_ initializing the instance, in order to subclass this `TestModel` class that we want to perform authorization over. But we need to provide the `checked_permissions` map (enumerating every action-resource pair we want to authorize) at `AuthorizedSQLAlchemy` initialization time, so... conundrum.